### PR TITLE
fix: make transaction-generating scripts more robust

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

Title says it all.

### Context of Change

There were some issues when working on price oracle.

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)

### Did you update HISTORY.md?

- [x] No, this change does not impact library users

## Test Plan

Tested locally.

### CODE REVIEW

**Comment by justinr1234** [comment]
@coderabbit full review

**Comment by justinr1234** [comment]
@coderabbitai full review

**Comment by mvadari** [comment]
https://github.com/coderabbitai full review

**Comment by mvadari** [comment]
@coderabbitai full review

**Review by justinr1234 — APPROVED** [review]

File: packages/xrpl/tools/generateModels.js [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 function processRippledSource(folder) {
   const sfieldCpp = readFile(
-    path.join(folder, 'src/ripple/protocol/impl/SField.cpp'),
+    path.join(folder, 'src/libxrpl/protocol/SField.cpp'),
```
[the remark this answers is not in the document]
Comment (mvadari) [reply packages/xrpl/tools/generateModels.js:18]: The files are in a different repo: https://github.com/XRPLF/rippled/

File: packages/xrpl/tools/generateModels.js [hunk 2]
```diff
@@ -49,10 +49,10 @@ function processRippledSource(folder) {
     ),
   )
   const transactionMatch = jsTransactionFile.match(
-    /export type Transaction =([| \nA-Za-z]+)\nexport/,
+    /export type SubmittableTransaction =([| \nA-Za-z]+)\n\/\*\*/,
```
[the remark this answers is not in the document]
Comment (mvadari) [reply packages/xrpl/tools/generateModels.js:52]: This regex is in the source code, so it is known to succeed. If it fails, the code needs to be updated.

**Review by pdp2121 — APPROVED** [review]

**Pull request merged**